skops/io/_numpy.py

```diff
@@ -49,6 +45,16 @@ def ndarray_get_state(obj: Any, save_state: SaveState) -> dict[str, Any]:
 
 
 def ndarray_get_instance(state, src):
+    if state["type"] == "base64":
+        # TODO
+        b64 = state["content"].encode("utf-8")
+        b = io.BytesIO(base64.b64decode(b64))
+        val = np.load(b)
+        if state["__class__"] != "ndarray":
+            cls = _import_obj(state["__module__"], state["__class__"])
+            val = cls(val)
+        return val
```

Comment:
why would we ever save in base64?

Reply:
This was a leftover from earlier, I'll remove it.